storage_file/wizards/replace_file.py

```diff
@@ -0,0 +1,28 @@
+# Copyright 2023 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+from odoo import api, models, fields
+
+class StorageFileReplace(models.TransientModel):
+    _name = "storage.file.replace"
+    _description = "TODO"
+    
+    file_id = fields.Many2one("storage.file")
+    new_file_id = fields.Many2one("storage.file")
+    data = fields.Binary()
+    file_name = fields.Char()
+
+    @api.onchange("data")
+    def onchange_data(self):
```

Comment:
why on onchange? This is going to cause the creation of tons of orphan files IMO